arch/lkl/kernel/threads.c

```diff
@@ -5,7 +5,7 @@
 #include <asm/cpu.h>
 #include <asm/sched.h>
 
-static volatile int threads_counter;
```

Comment:
I think this PR is a bit outdated: need to update/rebase in advance to apply the latest lkl.